coralproject/talk
# pluralize asset endpoints

## What does this PR do?

Pluralizes `asset` endpoints.

## How do I test this PR?

- everything should work still
- `npm run test`
- make sure I didn't miss anything
